app/__init__.py

```diff
@@ -14,7 +14,8 @@ def create_app():
     app = Flask(__name__)
     app.config.from_object(Config)
     app.secret_key = APP_SECRET_KEY
-    CORS(app, resources={r"/api/*": {"origins": "*"}})
+    # TODO: Include cloud target URL in origins field once project is hosted remotely
+    CORS(app, resources={r"/api/*": {"origins": "http://localhost:3000"}})
```

Comment:
set origins to either `'*'` or import from the env file `{env.get({'BASE_URL'})`